zeebe/protocol/src/main/java/io/camunda/zeebe/protocol/ColumnFamilyScoped.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright © 2017 camunda services GmbH (info@camunda.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.camunda.zeebe.protocol;
+
+public interface ColumnFamilyScoped {
```

Comment:
:upside_down_face: Should be `ScopedColumnFamily`, otherwise it sounds like something is scoped to just one column family, which is of course always the case.